devito/passes/iet/definitions.py

```diff
@@ -119,7 +119,7 @@ def _alloc_array_on_global_mem(self, site, obj, storage):
 
         # Create input array
         name = '%s_init' % obj.name
-        initvalue = np.array([pow_to_mul(i) for i in obj.initvalue])
+        initvalue = np.array([unevaluate(pow_to_mul(i)) for i in obj.initvalue])
```

Comment:
That doesn't make sense. `pow_to_mul` is already returning and unevaluated.

Reply:
only Muls